### PR TITLE
qownnotes: 20.2.5 -> 21.3.2

### PR DESCRIPTION
**Pull request by totoroot**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by totoroot** [comment]
I noticed that this package hasn't seen many updates and was therefore already pretty outdated.
If that's fine for @dtzWill I would like offer to take over maintenance of this package as I have used it for quite some time and follow the progress of the official repository of qownnotes closely.
I'm running NixOS 20.09 on my main machine and have the opportunity to test packages on a MacOS machine as well as other Linux VMs.

File: pkgs/applications/office/qownnotes/default.nix [hunk 1]
```diff
@@ -1,19 +1,19 @@
-{  mkDerivation, lib, fetchurl, qmake, qttools, qtbase, qtsvg, qtdeclarative, qtxmlpatterns, qtwayland, qtwebsockets, stdenv /* for isLinux */ }:
+{  mkDerivation, lib, fetchurl, qmake, qttools, qtbase, qtsvg, qtdeclarative, qtxmlpatterns, qtwayland, qtwebsockets, stdenv, qtx11extras /* for isLinux */ }:
```
Comment (SuperSandro2000) [inline pkgs/applications/office/qownnotes/default.nix:1]:
```suggestion
{  mkDerivation, lib, fetchurl, qmake, qttools, qtbase, qtsvg, qtdeclarative, qtxmlpatterns, qtwayland, qtwebsockets, stdenv, qtx11extras }:
```

**Comment by SuperSandro2000** [comment]
> I noticed that this package hasn't seen many updates and was therefore already pretty outdated.
> If that's fine for @dtzWill I would like offer to take over maintenance of this package as I have used it for quite some time and follow the progress of the official repository of qownnotes closely.
> I'm running NixOS 20.09 on my main machine and have the opportunity to test packages on a MacOS machine as well as other Linux VMs.

Let us wait a couple of days and if the other maintainer does not respond just add yourself as a maintainer. :)

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116983` at 65b9f555 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/4ee9b3ff978333469ae34547d0c5e832">qownnotes</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/applications/office/qownnotes/default.nix:24:5:
```
   |
24 |     license = licenses.gpl2;
   |     ^
```
</li>
  </ul>
</details>

**Comment by totoroot** [comment]
As @dtzWill didn't respond, I went ahead and added myself as a maintainer. Hope that is fine by you, @SuperSandro2000

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -10936,4 +10936,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  totoroot = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:10939]: Please sort this entry alphabetically.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Also please squash all the package related commits.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please squash all package and all maintainers commit into one commit each following the contributing guide.

**Comment by totoroot** [comment]
> Please squash all package and all maintainers commit into one commit each following the contributing guide.

@SuperSandro2000 Sorry, I didn't come around to finishing it up until today. The commits are squashed now and I believe it should fit the contributing guidelines now. Thanks for your efforts! I'll try to get into reviewing PRs soon too ✌️

**Comment by SuperSandro2000** [comment]
> Sorry, I didn't come around to finishing it up until today.

I am not waiting on this PR so take your time.

**Pull request merged**